app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -663,50 +655,6 @@ private void checkFavorite() {
                 }, error -> Log.e(TAG, Log.getStackTraceString(error)));
     }
 
-    void playExternalMedia(Intent intent, MediaType type) {
-        if (intent == null || intent.getData() == null) {
-            return;
-        }
-        if (Build.VERSION.SDK_INT >= 23
-                && ContextCompat.checkSelfPermission(this, Manifest.permission.READ_EXTERNAL_STORAGE)
-                != PackageManager.PERMISSION_GRANTED) {
-
-            if (ActivityCompat.shouldShowRequestPermissionRationale(this, Manifest.permission.READ_EXTERNAL_STORAGE)) {
-                Toast.makeText(this, R.string.needs_storage_permission, Toast.LENGTH_LONG).show();
-            }
-
-            int code = REQUEST_CODE_STORAGE_PLAY_AUDIO;
```

Comment:
Could you please remove the `REQUEST_CODE_STORAGE_PLAY_AUDIO` variable, too?

Reply:
I've removed the unused variable. But due to some reason the unit test keep timing out on my local machine.

Reply:
Looks good to me. The tests are sometimes a bit unreliable. I just restarted them.